### ISSUE TITLE
Multiselect: Accessibility issue `aria-allowed-role` - list-box role not allowed

### ISSUE THREAD

**Issue by shreyashgharod** [primefaces/primereact#5446]
### Describe the bug

I am facing a accessibility issue while using prime react component <Multiselect> for multiple selection dropdown. 
" aria-allowed-role - list-box role not allowed "
![DD ISSUE](https://github.com/primefaces/primereact/assets/56091591/17618bb3-f20c-46db-8393-d7040af6b189)

This similar issue is on prime react website also . If Fast Pass (Accessibility Insights for Web - this is a chrome extension) is run on prime react website.
![PREACT ISSUE](https://github.com/primefaces/primereact/assets/56091591/9c2cbdfd-8828-498a-bb9b-e4e28e1ca309)

Is there any fix to resolve this issue . Please do contribute. 

You can check the working code in Sandbox : [ https://codesandbox.io/p/sandbox/primereact-multiselect-forked-wtpshd?file=%2Fsrc%2FApp.js ](url)

### Reproducer

https://codesandbox.io/p/sandbox/primereact-multiselect-forked-wtpshd?file=%2Fsrc%2FApp.js

### PrimeReact version

9.6.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome

### Steps to reproduce the behavior

1. Go to prime react website & open Multiselect Component page  [https://primereact.org/multiselect/](url) in Google Chrome
2. Open Accessibility Insights for Web extension and run fast pass . ( if you don't have the extension add from chrome extensions)
3. You will see red box with exclamation on the Multiselect component.
4. Click on it and see the issue. 
![DD ISSUE](https://github.com/primefaces/primereact/assets/56091591/33e2543d-83d4-4782-a9a1-83f7797def5b)
![PREACT ISSUE](https://github.com/primefaces/primereact/assets/56091591/baaddaa5-ed8a-4a78-b123-77ed7067ee7b)

### Expected behavior

There should be no accessibility issue.

**Comment by melloware** [comment]
@shreyashgharod can you check PrimeVue: https://primevue.org/multiselect/   PrimeReact will be getting all the accesibility fixes from PrimeVue so check if its an issue there also?
